lib/app/data.dart

```diff
@@ -81,17 +93,71 @@ class User implements Entity {
   final List<Id<Role>> roles;
   bool get isTeacher => hasRole(Role.teacherName);
   bool hasRole(String name) {
-    // TODO(marcelgarus): Remove the hard-coded mapping and use runtime lookup when upgrading flutter_cached and flattening is supported
+    // TODO(marcelgarus): Remove the hard-coded mapping and use runtime lookup
+    // when upgrading flutter_cached and flattening is supported.
```

Comment:
TODO-suggestions from linters don't work when comments are broken across lines
```suggestion
    // TODO(marcelgarus): Remove the hard-coded mapping and use runtime lookup when upgrading flutter_cached and flattening is supported.
```